### DISCUSSION TITLE
Build private docker registry to improve the speed of pulling Milvus Image

### DISCUSSION THREAD

**Discussion by Renekton** [milvus-io/milvus#16670] [Ideas & Feature requests]
As we all, the speed of pulling image from the official docker registry is instability in China. 
Although sometimes we could use some accelerators which is not convenient enough for someone who is not familiar with docker.  The effect of accelerator also is not stable.
I think this problem has a bad influence on the promotion of milvus.

Maybe we can do something to improve the speed of pulling Milvus Docker Image in China. such as :

1. Build private docker registry to improve the speed of pulling Milvus Image. We can consider this solution : https://help.aliyun.com/product/60716.html?spm=5176.22414175.sslink.2.10a12e75Z9GWWS

2. Use the tar image file which produced by the command "docker save" and then docker load the image file. Maybe we need to consider the safety of the file. The file can be stored in oss, this solution is inexpensive.

**Comment by jennyli-z** [comment]
For the one who is not familiar with docker, probably use apt or yum is the  more convenient way to install milvus...https://milvus.io/docs/v2.0.x/install_standalone-aptyum.md

**Reply by Renekton** [reply]
apt or yum is not good option in devops. I want to improve the speed of pulling image.  The effect of accelerator also is not stable.
